amp/repository/aim/view/changePassword.jsp

```diff
@@ -105,10 +105,10 @@ function validate(){
 						</digi:trn>
 					</td>
 					<td align="left" style="padding-bottom:10px;">
-						<c:if test="${!umViewEditUserForm.getUserIdEnabled}">
+						<c:if test="${umViewEditUserForm.getUserIdEnabled}">

```

Comment:
the form instance name is not umViewEditUserForm is 
aimChangePasswordForm
you need to access it as a property userIdEnabled instead of getUserIdEnabled


also you can try to use this code that is cleaner

`                        <html:text property="userId" styleClass="pwd_username"
                                   readonly="${aimChangePasswordForm.userIdEnabled}"
                                   size="20"/><span>&nbsp;&nbsp;&nbsp;</span>`